django/db/backends/postgresql/introspection.py

```diff
@@ -22,15 +23,45 @@ class DatabaseIntrospection(BaseDatabaseIntrospection):
         700: 'FloatField',
         701: 'FloatField',
         869: 'GenericIPAddressField',
+        1000: ('django.contrib.postgresql.fields.ArrayField',
```

Comment:
I don't like the idea of contrib.postgres getting special treatment in the backend. If some third-party app implements these fields, I think they should have a blessed way of "contributing" them to the introspection rules which contrib.postgres should also use.